package/src/components/MessageList/InlineDateSeparator.tsx

```diff
@@ -22,29 +22,41 @@ const styles = StyleSheet.create({
   },
 });
 
+/**
+ * Props for the `InlineDateSeparator` component.
+ */
 export type InlineDateSeparatorProps = {
+  /**
+   * Date to be displayed.
+   */
   date?: Date;
+  /*
+   * Lookup key in the language corresponding translations sheet to perform date formatting
+   */
+  timestampTranslationKey?: string;
```

Comment:
is prop necessary? its already customisable

Reply:
This customizes the key in the JSON.